test/test_quantization.py

```diff
@@ -1070,6 +1071,15 @@ def test_observer_scriptable(self):
         loaded = torch.jit.load(buf)
         self.assertEqual(obs.calculate_qparams(), loaded.calculate_qparams())
 
+    def test_no_qconfig_propagation(self):
+        model = ModelWithNoQconfigPropagation()
+        model.qconfig = torch.quantization.default_qconfig
+
+        model = prepare(model)
+        self.assertTrue(hasattr(model.fc1, 'qconfig'), "QConfig not propagating")
```

Comment:
Please make the error message more informative, something like :"Expect Qconfig to propagate to linear module"